docs/book/basics/status_subresource.md

```diff
@@ -0,0 +1,127 @@
+## Status Subresource 
+By convention, the Kubernetes API makes a distinction between the specification
+of the desired state of an object (a nested object field called "spec") and the 
+status of the object at the current time (a nested object field called "status"). 
+The specification is a complete description of the desired state, including 
+configuration settings provided by the user, default values expanded by the 
+system, and properties initialized or otherwise changed after creation by other 
+ecosystem components (e.g., schedulers, auto-scalers), and is persisted in stable
```

Comment:
Maybe change `stable storage` to `etcd` to avoid confusing users with persistent volumes.

Reply:
Done.